EventFilter/SiPixelRawToDigi/plugins/SiPixelDigiErrorsFromSoA.cc

```diff
@@ -48,12 +49,12 @@ class SiPixelDigiErrorsFromSoA : public edm::stream::EDProducer<> {
 };
 
 SiPixelDigiErrorsFromSoA::SiPixelDigiErrorsFromSoA(const edm::ParameterSet& iConfig)
-    : digiErrorSoAGetToken_{consumes<SiPixelDigiErrorsSoA>(iConfig.getParameter<edm::InputTag>("digiErrorSoASrc"))},
+    : cablingToken_(esConsumes(edm::ESInputTag(iConfig.getParameter<std::string>("CablingMapLabel")))),
```

Comment:
Based on a few other examples I have seen, I believe this should be

```suggestion
    : cablingToken_(esConsumes(edm::ESInputTag("", iConfig.getParameter<std::string>("CablingMapLabel")))),
```

The same applies to the other similar examples in this PR: